src/main/java/org/generationcp/middleware/service/impl/GermplasmGroupingServiceImpl.java

```diff
@@ -117,8 +118,26 @@ public GermplasmGroup getGroupMembers(final Germplasm founder) {
 	}
 
 	@Override
-	public List<Germplasm> getGroupMembers(final Integer gid) {
-		return this.germplasmDAO.getManagementGroupMembers(gid);
+	public List<Germplasm> getDescendantGroupMembers(final Integer gid, final Integer mgid) {
+		final List<Germplasm> descendantGroupMembers = new ArrayList<>();
+		this.populateDescendantGroupMembers(descendantGroupMembers, gid, mgid);
+		return descendantGroupMembers;
+	}
+
+	private void populateDescendantGroupMembers(final List<Germplasm> descendantGroupMembers, final Integer gid, final Integer mgid) {
+		// Get advanced children
```

Comment:
Why are we getting only non-generative children? It is not clear from the service name that generative children will be excluded. Maybe we can include in method signature: 
List<Germplasm> getDescendantGroupMembers(Integer gid, Integer mgid, final List<MethodType> filterMethodTypes)

Reply:
Hi @darla-leafnode, code inheritance are only applied to advanced children.